app/model/view/InstanceViewFactory.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+namespace App\Model\View;
+
+use App\Helpers\Localizations;
+use App\Model\Entity\Instance;
+use App\Model\Entity\LocalizedGroup;
+
+
+/**
+ * Factory for instance views which somehow do not fit into json serialization
+ * of entities.
+ */
+class InstanceViewFactory {
+
+  /** @var GroupViewFactory */
+  private $groupViewFactory;
+
+  public function __construct(GroupViewFactory $groupViewFactory) {
+    $this->groupViewFactory = $groupViewFactory;
+  }
+
+
+  /**
+   * Get as much instance detail info as your permissions grants you.
+   * @param Instance $instance
+   * @return array
+   */
+  public function getInstance(Instance $instance): array {
+    /** @var LocalizedGroup $localizedRootGroup */
+    $localizedRootGroup = Localizations::getPrimaryLocalization($instance->getRootGroup()->getLocalizedTexts());
+
+    return [
+      "id" => $instance->getId(),
+      "name" => $localizedRootGroup ? $localizedRootGroup->getName() : "",  # BC
+      "description" => $localizedRootGroup ? $localizedRootGroup->getDescription() : "",  # BC
```

Comment:
Is there a reason, why is BC comment started with `#` instead of `//`?

Reply:
No there is not